specification/specs/x_pack/data_streams/get_data_stream/IndicesGetDataStreamResponse.ts

```diff
@@ -18,6 +18,26 @@
  */
 
 class IndicesGetDataStreamResponse extends ResponseBase {
-  stub: integer
-  // https://www.elastic.co/guide/en/elasticsearch/reference/7.12/indices-get-data-stream.html#get-data-stream-api-response-body
+  data_streams: IndicesGetDataStreamItem[]
+}
+
+class IndicesGetDataStreamItem {
+  name: DataStreamName
+  timestamp_field: IndicesGetDataStreamItemTimestampField
+  indices: IndicesGetDataStreamItemIndex[]
+  generation: integer
+  template: Name
+  hidden: boolean
+  status: string // TODO: use type Health --> must be case insensitive
```

Comment:
You can define the enum like we are doing in the query dsl:

https://github.com/elastic/elastic-client-generator/blob/174b5bd7c8be68b78e64f8f38efcabaef78a6119/specification/specs/query_dsl/Operator.ts#L20-L25

Reply:
Added `DataStreamHealthStatus` with the aliases